RadixWallet/Clients/SecurityCenterClient/SecurityCenterClient+Interface.swift

```diff
@@ -0,0 +1,49 @@
+import CloudKit
+import ComposableArchitecture
+import DependenciesAdditions
+import os
+
+// MARK: - SecurityCenterClient
+public struct SecurityCenterClient: DependencyKey, Sendable {
+	public let problems: Problems
+	public let lastManualBackup: LastManualBackup
+	public let lastCloudBackup: LastCloudBackup
+}
+
+// MARK: SecurityCenterClient.Problems
+extension SecurityCenterClient {
+	public typealias Problems = @Sendable () async -> AnyAsyncSequence<[SecurityProblem]>
+	public typealias LastManualBackup = @Sendable () async -> AnyAsyncSequence<BackupStatus>
+	public typealias LastCloudBackup = @Sendable () async -> AnyAsyncSequence<BackupStatus>
+}
+
+// MARK: - SecurityProblem
+public enum SecurityProblem: Hashable, Sendable, Identifiable {
+	case problem3(accounts: Int, personas: Int)
```

Comment:
```suggestion
	/// User has at least one shield, but has no XRD. User has an account (or accounts) without a shield applied, meaning it is only secured by a single factor.
	/// That factor (as is often, but not always the case) is a phone key factor and the user has never viewed the seed phrase and confirmed that they wrote it down.
	/// Losing the phone means losing that account forever.
	case problem3(accounts: Int, personas: Int)
```

This is what I meant @kugel3

Reply:
I know, but it's better to link it, since that Confluence page might update in the future

Reply:
Right, but if that confluence page updates and the implementation doesn't, then the linked Confluence will have logic that doesn't correspond to what we have on code. That's why I believe it is better to keep the link on top, but also reference on each case so that any developer can see what `problemX` means in code.